.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for Trino Python client
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+### Describe the feature
+A clear and concise description of what you want to happen.
+
+### Describe alternatives you've considered
+A clear and concise description of any alternative solutions or features you've considered.
+
+### Additional context
+Please include any other relevant context here.
+
+### Who will this benefit?
```

Comment:
Obviously the author :-) I think it's too vague, a problem statement and a solution should be sufficient.

Reply:
I think this may let the author think more about which persona will benefit from the feature and leave less to the imagination of the person producing the PR.

Reply:
But what would you use that information for?

Reply:
I agree it does not necessarily need a separate section but knowing who the feature is for, is part of a good problem statement. It will help people detect gaps in the feature or may trigger more reflection.